### DISCUSSION TITLE
Common Functions

### DISCUSSION THREAD

**Discussion by robbrad** [robbrad/UKBinCollectionData#102] [General]
@dp247 : what are your thoughts on including the common functions in the abstract class instead of a separate file?

This would be in keeping with the design pattern mentioned in https://github.com/robbrad/UKBinCollectionData/blob/master/CONTRIBUTING.md

Maybe it makes sense to rename the get_bin_data.py

**Comment by dp247** [comment]
I'm all for it. It would save on the additional import too.
